kernel/src/common/regs/mod.rs

```diff
@@ -237,6 +253,16 @@ impl<R: RegisterLongName> FieldValue<u32, R> {
             associated_register: PhantomData,
         }
     }
+
+    pub fn to_u32(&self) -> u32 {
```

Comment:
I personally consider `to_u32` harmful. I think if you're using it, you're losing some of the advantages of having this whole register system which checks validity for you. For every case above there is a way that we can use the register system without this extension. Therefore, I recommend that it not be added.

Reply:
I agree -- I just wasn't sure how to get around it. But I think your code will do it, I'll try.

Reply:
Can we remove these now? I believe they are unused.